.aws/deploy/support-task-definition.prod.json

```
@@ -0,0 +1,304 @@
+        },
+        {
+          "name": "SGID_REDIRECT_URI",
+          "valueFrom": "PROD_SGID_REDIRECT_URI"
+        },
+        {
+          "name": "SGID_PRIVATE_KEY",
+          "valueFrom": "PROD_SGID_PRIVATE_KEY"
+        },
+        {
+          "name": "SITE_CLONE_FORM_KEY",
+          "valueFrom": "PROD_SITE_CLONE_FORM_KEY"
+        },
+        {
+          "name": "SITE_CREATE_FORM_KEY",
+          "valueFrom": "PROD_SITE_CREATE_FORM_KEY"
+        },
+        {
+          "name": "SITE_LAUNCH_DYNAMO_DB_TABLE_NAME",
+          "valueFrom": "PROD_SITE_LAUNCH_DYNAMO_DB_TABLE_NAME"
+        },
+        {
+          "name": "SITE_LAUNCH_FORM_KEY",
+          "valueFrom": "PROD_SITE_LAUNCH_FORM_KEY"
+        },
+        {
+          "name": "SITE_PASSWORD_SECRET_KEY",
+          "valueFrom": "PROD_SITE_PASSWORD_SECRET_KEY"
+        },
+        {
+          "name": "PROD_SSH_PUBLIC_KEY",
+          "valueFrom": "PROD_SSH_PUBLIC_KEY"
+        },
+        {
+          "name": "PROD_SSH_PRIVATE_KEY",
+          "valueFrom": "PROD_SSH_PRIVATE_KEY"
+        },
+        {
+          "name": "STEP_FUNCTIONS_ARN",
+          "valueFrom": "PROD_STEP_FUNCTIONS_ARN"
+        },
+        {
+          "name": "SYSTEM_GITHUB_TOKEN",
+          "valueFrom": "PROD_SYSTEM_GITHUB_TOKEN"
+        },
+        {
+          "name": "UPTIME_ROBOT_API_KEY",
+          "valueFrom": "PROD_UPTIME_ROBOT_API_KEY"
+        },
+        {
+          "name": "SITE_CHECKER_FORM_KEY",
+          "valueFrom": "PROD_SITE_CHECKER_FORM_KEY"
+        },
+        {
+          "name": "SITE_AUDIT_LOGS_FORM_KEY",
+          "valueFrom": "PROD_SITE_AUDIT_LOGS_FORM_KEY"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-staging-node18/var/log/web.stdout.log",
+          "awslogs-region": "ap-southeast-1",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "dd-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "portMappings": [
+        {
+          "containerPort": 8126,
+          "hostPort": 8126,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [
+        {
+          "name": "TZ",
+          "value": "Asia/Singapore"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        },
+        {
+          "name": "ECS_FARGATE",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.com"
+        },
+        {
+          "name": "DD_ENV",
+          "value": "production"
+        },
+        {
+          "name": "DD_SERVICE",
+          "value": "isomer-support"
+        },
+        {
+          "name": "DD_TAGS",
+          "value": "team:isomer"
```

/notblocker: In case we are not seeing the service reported correctly, the service tag can be added here too (we've seen the infra linkage didn't work without it for armoury):
```suggestion
          "value": "team:isomer,service:isomer-support"
```

---

i actually removed cos i thought it might conflict iwth the `DD_SERVICE` tag @__@ good to know, will add in!

---

updated